.github/labels.js

```diff
@@ -66,6 +66,31 @@ module.exports = [
         color: "912E29",
         description: ""
     },
+    {
+        name: "priority:blocker",
+        color: "cc0000",
+        description: "Blocks development and/or testing work, production could not run"
```

Comment:
Hmm, I've been interpreting this as blocks release. That is, don't release before this is fixed.

Reply:
Yes you think in good way, we have:
> check existing issues with Blocker priority, consider to fix it or change priority if reasonable

https://maven.apache.org/developers/release/maven-project-release-procedure.html